build/Makefile

```diff
@@ -51,13 +51,14 @@ endef
 # run when all is used, but we want it to work
 $(foreach t,$(DEFAULT_TARGETS) $(ADDITIONAL_TARGETS),$(eval $(call per_target,$(t))))
 
+SUPPRESS_ERROR_OUTPUT := 2> /dev/null
 clean: start
 	@echo 'Cleaning...'
-	-@rm -Rf $(BIN_DIR)
-	-@find $(ROOT_DIR) -path '*/__pycache__/*' -exec rm {} \;
-	-@find $(ROOT_DIR) -name __pycache__ -exec rmdir {} \;
-	-@find $(ROOT_DIR) -name '*.pyc' -exec rm {} \;
-	-rm $(foreach d,$(DRIVERS), $(ROOT_DIR)/docs/$(d)/*.rst)
+	-@rm -Rf $(BIN_DIR) $(SUPPRESS_ERROR_OUTPUT)
```

Comment:
This may not work with find on Windows either. I think the parameters are different.